content/design/color/index.html

```diff
@@ -2,58 +2,440 @@
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-info</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#00b4f1</div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #71bf43"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Highlight for elements conveying an "success" status, such as alerts and toasts</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-success</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#71bf43</div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #fbb034"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Highlight for elements conveying an "warning" status, such as alerts, labels and toasts</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-warning</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#fbb034</div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #ef4044"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Highlight for elements conveying an "danger" status, such as alerts, labels and toasts</div>
+      </div>
+      <div class="definition-code">
+        <div class="bb-field-label">Code</div>
+        <code>$sky-highlight-color-danger</code>
+      </div>
+      <div class="definition-styling">
+        <div class="bb-field-label">Styling</div>
+        <div>#ef4044</div>
+      </div>
+    </div>
+  </div>
+  
+  <h2 class="bb-section-heading">Borders</h2>
 
-{{ include 'includes/design-table.hbs' }}
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="swatch" style="background-color: #e2e3e4"></div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="bb-field-label">Usage</div>
+        <div>Borders separating sections of content within a container</div>
```

Comment:
I'd change "separating" to "that separate" here and in the next description if we change "conveying" to "that convey" in the previous descriptions.